src/gui/Src/Bridge/Bridge.cpp

```diff
@@ -33,6 +33,7 @@ void Bridge::CopyToClipboard(const QString & text)
 {
     QClipboard* clipboard = QApplication::clipboard();
     clipboard->setText(text);
+    GuiAddStatusBarMessage(QT_TRANSLATE_NOOP("DBG", "The data has been copied to clipboard.\n"));

```

Comment:
This will not translate it 😄 The correct way is `tr("The data has been copied to clipboard.\n").toUtf8().constData()`